lib/widget/form_textfield.dart

```diff
@@ -287,7 +298,7 @@ class TextInputState extends FormFieldWidgetState<BaseTextInput> {
         return null;
       },
       textInputAction: widget._controller.keyboardAction,
-      keyboardType: widget.keyboardType,
```

Comment:
why are you removing keyboardType? This is important to show the correct keyboard depending on the input.

Reply:
oh sorry line got deleted by mistake